フラッシュメッセージの表示機能の実装

Kaoru1126 opened the pull request:
# WHAT
フラッシュメッセージの表示機能の実装
- サインアップ時、ログイン時、ログアウト時にフラッシュメッセージの表示
- 上記メッセージのローカライズ

# WHY
ChatSpaceの開発に必要だから

Kaoru1126 committed:
create pull request

Kaoru1126 committed:
changed local lang to JPN

Kaoru1126 committed:
adding view file for flash msg ans set scss

Kaoru1126 committed:
add the render to load the flash msg

Kaoru1126 committed:
omitted unnecessary head_bar

yusukeyamane commented on app/assets/stylesheets/application.scss:
```diff
@@ -3,6 +3,7 @@
 @import "./reset";
```
今回のPRには関係ないのですが、_variables.scssは読み込んで居なくてもカラーの読み込みは行われていますか？

Kaoru1126 replied:
"./reset"が読み込まれる順番を変えました。カラーは読み込まれています。

Kaoru1126 replied:
疑問なんですが、外部ファイルを上から順番に読み込んでて、resetがvariablesより下にあった時もカラーが読み込まれてたんですけど、これってどうしてカラーが読み込まれてたのでしょうか？

yusukeyamane commented:
フラッシュメッセージのスクリーンショットをお願いします

Kaoru1126 committed:
omitted unnecessary lines

Kaoru1126 committed:
changed orders for files imported

plus, omitted unnecessary lines

Kaoru1126 commented:
フラッシュメッセージの画像を添付しました。
ご確認お願い致します。
<img width="1244" alt="log_in" src="https://user-images.githubusercontent.com/19361155/26999861-7acf40b0-4dde-11e7-824c-36bb8c82db58.png">
<img width="1243" alt="log_out" src="https://user-images.githubusercontent.com/19361155/26999862-7ad2dc34-4dde-11e7-8371-6789e5c40b33.png">
<img width="1243" alt="signed_in" src="https://user-images.githubusercontent.com/19361155/26999863-7ad4d926-4dde-11e7-85be-9787749f82d9.png">

Kaoru1126 committed:
added Class to alert_alert-notice

Kaoru1126 committed:
testing id color works or not

Kaoru1126 committed:
omitted unnecessary lines

Kaoru1126 committed:
changed order again.

wondering why color codes still work when the order is like this. it
should not be working since the reset  file is below the variables file…
